api/openapi.gen.json

```diff
@@ -66,7 +66,7 @@
           "amount": 1,
           "image_id": "composer-api-081fc867-838f-44a5-af03-8b8def808431",
           "instance_size": "Basic_A0",
-          "location": "useast_1",
+          "location": "useast",
```

Comment:
I forgot to update this, I guess this speaks the main message what most of this patch does.